packages/selection/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import { Accessor, createEffect, createSignal, onCleanup, Setter } from "solid-js";
+
+export type HTMLSelection = [node: HTMLElement | null, start: number, end: number];
+
+export const getTextNodes = (node: Node) => {
```

Comment:
If you are exporting this function, maybe would it be worth it to mention it in the readme, as an additional *vanilla* utility?

Reply:
I was exporting it mostly for testing purposes. If we're in need of such a function in other priitives, that's when I would move it to utils.